.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+
+_Please include a summary of the changes, relevant motivation and context_.
+
+# Issue
+
+* _Redmine Ticket URL_ or _Github Issue ID_.
```

Comment:
I think we would not really want to post Redmine tickets anymore, only Github issues. Our Redmine tickets would be the ones having the reference to the issue/PR.